sgf-viewer/public/sgfs/10mil_visits/game-0.sgf

```diff
@@ -0,0 +1 @@
+(;FF[4]CA[UTF-8]AP[gogui-twogtp:1.5.1]KM[6.5]PB[Victim]PW[Adversary]DT[2023-01-13]RE[W+R]C[Black command: /engines/KataGo-raw/cpp/katago gtp -config /go_attack/configs/gtp-raw.cfg -config go_attack/configs/compute/7gpu.cfg -model /shared/victims/kata1-b40c256-s11840935168-d2898845681.bin.gzWhite command: /engines/KataGo-custom/cpp/katago gtp -config /go_attack/configs/gtp-amcts.cfg -config go_attack/configs/compute/7gpu.cfg -model /shared/victimplay/ttseng-avoid-pass-alive-coldstart-39-20221025-175949/models/t0-s545065216-d136760487/model.bin.gz -victim-model /shared/victims/kata1-b40c256-s11840935168-d2898845681.bin.gzBlack version: 1.12.2White version: 1.10.0Result[Black\]: W+RResult[White\]: W+RHost: go-gtp-ttseng-v10mil-launch-0-0 (AMD EPYC 7763 64-Core Processor)Date: January 14, 2023 at 5:01:05 AM UTC];B[pd];W[oe];B[pe];W[od];B[oc];W[nb];B[nc];W[md];B[mc];W[ld];B[lc];W[nd];B[kd];W[of];B[qg];W[cc];B[ph];W[qq];B[ke];W[le];B[kg];W[lf];B[kf];W[cq];B[mh];W[ng];B[oi];W[qb];B[pc];W[lg];B[lh];W[mg];B[ed];W[bf];B[dp];W[dq];B[dc];W[cb];B[qo];W[pb];B[oo];W[ob];B[rc];W[rb];B[sb];W[qc];B[rd];W[nr];B[fp];W[gr];B[cf];W[be];B[eq];W[cp];B[dn];W[bn];B[cd];W[bd];B[jo];W[kq];B[oq];W[or];B[kp];W[kr];B[pq];W[pr];B[rq];W[qr];B[ik];W[rg];B[ch];W[db];B[fc];W[rh];B[rj];W[eb];B[dk];W[fb];B[mp];W[er];B[hc];W[bg];B[hp];W[bk];B[di];W[qk];B[qi];W[jc];B[gb];W[jb];B[kc];W[rk];B[rn];W[sj];B[ri];W[bi];B[cm];W[sk];B[ce];W[si];B[rp];W[rr];B[lq];W[lr];B[co];W[oj];B[qf];W[bo];B[gq];W[fr];B[iq];W[pj];B[ni];W[ol];B[qm];W[hr];B[nq];W[mr];B[ec];W[bm];B[cl];W[ir];B[cj];W[jd];B[jq];W[jh];B[kh];W[bj];B[cg];W[jr];B[bl];W[ij];B[bh];W[al];B[am];W[ml];B[jk];W[hh];B[jj];W[an];B[ah];W[gg];B[ig];W[hk];B[ii];W[hi];B[fq];W[hj];B[ji];W[gl];B[hm];W[hf];B[ie];W[ih];B[je];W[fh];B[fk];W[he];B[gi];W[id];B[mb];W[nj];B[pl];W[na];B[cn];W[lm];B[pk];W[if];B[jg];W[ma];B[hd];W[kb];B[ga];W[jm];B[qp];W[kl];B[lk];W[il];B[mk];W[kk];B[ki];W[kj];B[mj];W[lj];B[ak];W[aj];B[ai];W[lb];B[bp];W[bq];B[ck];W[gm];B[hn];W[hl];B[la];W[hg];B[qd];W[rf];B[nm];W[re];B[nl];W[qj];B[qh];W[ll];B[bb];W[bc];B[sr];W[ba];B[fa];W[nk];B[mi];W[pf];B[qe];W[oh];B[pi];W[pg];B[nh];W[jl];B[mq];W[li];B[ea];W[jf];B[sc];W[sd];B[se];W[ra];B[og];W[sa];B[sf];W[oh];B[ao];W[ap];B[og];W[sq];B[sh];W[sg];B[sh];W[sg];B[pa];W[rh];B[ci];W[rf];B[sp];W[re];B[ag];W[sd];B[se];W[rg];B[qa];W[ka];B[];W[oh];B[];W[ok];B[og];W[pm];B[jp];W[ql];B[so];W[oh];B[om];W[pn];B[og];W[rm];B[];W[oh];B[qn];W[sd])
```

Comment:
Note this leaks your username. OK for public version but we'll need to strip this out for double blind I think?

Reply:
good, point, I search-and-replaced `ttseng` with `anonymized`